standard_reports/outbound_shipments/latest/src/template.html

```diff
@@ -26,23 +21,23 @@ <h3>{{data.otherPartyName}}</h3>
     <tbody>
       {% for line in data.lines %}
       <tr>
+        <td>{{line.otherPartyName}}</td>
```

Comment:
nit: seeing as it is sorted by item and then customer, I think I'd expect this column after the item name 👀